lib/pg_query/pg_query_pb.rb

```diff
@@ -5,7 +5,7 @@
\x18\x06 \x03(\x0b\x32\x0e.pg_query.NodeR\x14indexIncludingParams\x12(\n\x07options\x18\x07 \x03(\x0b\x32\x0e.pg_query.NodeR\x07options\x12\x31\n\x0cwhere_clause\x18\x08 \x01(\x0b\x32\x0e.pg_query.NodeR\x0bwhereClause\x12\x38\n\x10\x65xclude_op_names\x18\t \x03(\x0b\x32\x0e.pg_query.NodeR\x0e\x65xcludeOpNames\x12\x1e\n\nidxcomment\x18\n \x01(\tR\nidxcomment\x12\x1b\n\tindex_oid\x18\x0b \x01(\rR\x08indexOid\x12\x1d\n\nold_number\x18\x0c \x01(\rR\toldNumber\x12(\n\x10old_create_subid\x18\r \x01(\rR\x0eoldCreateSubid\x12\x43\n\x1eold_first_relfilelocator_subid\x18\x0e \x01(\rR\x1boldFirstRelfilelocatorSubid\x12\x16\n\x06unique\x18\x0f \x01(\x08R\x06unique\x12.\n\x12nulls_not_distinct\x18\x10 \x01(\x08R\x12nulls_not_distinct\x12\x18\n\x07primary\x18\x11 \x01(\x08R\x07primary\x12\"\n\x0cisconstraint\x18\x12 \x01(\x08R\x0cisconstraint\x12\x1e\n\ndeferrable\x18\x13 \x01(\x08R\ndeferrable\x12\"\n\x0cinitdeferred\x18\x14 \x01(\x08R\x0cinitdeferred\x12 \n\x0btransformed\x18\x15 \x01(\x08R\x0btransformed\x12\x1e\n\nconcurrent\x18\x16 \x01(\x08R\nconcurrent\x12$\n\rif_not_exists\x18\x17 \x01(\x08R\rif_not_exists\x12\x32\n\x14reset_default_tblspc\x18\x18 \x01(\x08R\x14reset_default_tblspc\"\xa9\x02\n\x0f\x43reateStatsStmt\x12*\n\x08\x64\x65\x66names\x18\x01 \x03(\x0b\x32\x0e.pg_query.NodeR\x08\x64\x65\x66names\x12.\n\nstat_types\x18\x02 \x03(\x0b\x32\x0e.pg_query.NodeR\nstat_types\x12$\n\x05\x65xprs\x18\x03 \x03(\x0b\x32\x0e.pg_query.NodeR\x05\x65xprs\x12,\n\trelations\x18\x04 \x03(\x0b\x32\x0e.pg_query.NodeR\trelations\x12\x1e\n\nstxcomment\x18\x05 \x01(\tR\nstxcomment\x12 \n\x0btransformed\x18\x06 \x01(\x08R\x0btransformed\x12$\n\rif_not_exists\x18\x07 \x01(\x08R\rif_not_exists\"C\n\tStatsElem\x12\x12\n\x04name\x18\x01 \x01(\tR\x04name\x12\"\n\x04\x65xpr\x18\x02 \x01(\x0b\x32\x0e.pg_query.NodeR\x04\x65xpr\"\x82\x01\n\x0e\x41lterStatsStmt\x12*\n\x08\x64\x65\x66names\x18\x01 \x03(\x0b\x32\x0e.pg_query.NodeR\x08\x64\x65\x66names\x12$\n\rstxstattarget\x18\x02 \x01(\x05R\rstxstattarget\x12\x1e\n\nmissing_ok\x18\x03 \x01(\x08R\nmissing_ok\"\xb9\x02\n\x12\x43reateFunctionStmt\x12\"\n\x0cis_procedure\x18\x01 \x01(\x08R\x0cis_procedure\x12\x18\n\x07replace\x18\x02 \x01(\x08R\x07replace\x12*\n\x08\x66uncname\x18\x03 \x03(\x0b\x32\x0e.pg_query.NodeR\x08\x66uncname\x12.\n\nparameters\x18\x04 \x03(\x0b\x32\x0e.pg_query.NodeR\nparameters\x12\x33\n\x0breturn_type\x18\x05 \x01(\x0b\x32\x12.pg_query.TypeNameR\nreturnType\x12(\n\x07options\x18\x06 \x03(\x0b\x32\x0e.pg_query.NodeR\x07options\x12*\n\x08sql_body\x18\x07 \x01(\x0b\x32\x0e.pg_query.NodeR\x08sql_body\"\xb5\x01\n\x11\x46unctionParameter\x12\x12\n\x04name\x18\x01 \x01(\tR\x04name\x12-\n\x08\x61rg_type\x18\x02 \x01(\x0b\x32\x12.pg_query.TypeNameR\x07\x61rgType\x12\x33\n\x04mode\x18\x03 \x01(\x0e\x32\x1f.pg_query.FunctionParameterModeR\x04mode\x12(\n\x07\x64\x65\x66\x65xpr\x18\x04 \x01(\x0b\x32\x0e.pg_query.NodeR\x07\x64\x65\x66\x65xpr\"\x9b\x01\n\x11\x41lterFunctionStmt\x12.\n\x07objtype\x18\x01 \x01(\x0e\x32\x14.pg_
```

Comment:
For context, this is because of a incorrect type in the Protobuf: https://github.com/pganalyze/libpg_query/commit/6f269718fb7095ac88a595d7d36390511d27297b

(because this message is not actually used in raw parsetrees its safe to change this in a minor release without flagging it in the changelog)